configs/models/google_gemma-2_9b_it.yaml

```diff
@@ -0,0 +1,7 @@
+model_name: google/gemma-2-9b-it
+trust_remote_code: false
+parallelize: false
+add_bos_token: false
```

Comment:
Thanks for adding these model files, for Gemma you might want to make `add_bos_token: false`

Reply:
https://github.com/EleutherAI/lm-evaluation-harness/pull/2049